AjobK/Athena
# 208 avatar upload

closes #208 

 **What is it supposed to do**
Allows user to change their avatars locally. Only .png and .jpg files are allowed and the file size must be below a specific amount of KB (can be adjusted). Forbidden inputs will result in an error pop up. Cropping is not yet implemented, so image resolutions are ignored. Users can also drop files (not intended but heh). Submitting the avatar sends the file to an invalid route for demonstration purposes. The avatar will also be changed on the profile, but these changes will disappear after you leave the page (I will work on this in the next sprint).

 **How can we test your branch**
Log in and go to your profile. Click on your avatar (hovering over it should show a pen icon) to open the file explorer. Try to select some files. The current max file size is 40 KB, so anything higher than that should not be allowed and an error pop up should appear. You can input forbidden file types by dragging a file from your file explorer into the avatar element on your profile, which should also fail. Submitting the avatar will give an error in console, as the route does not yet exist. It should still replace the current avatar with the new avatar though.

 **What broweser have been checked**
